tests/torch/sparsity/movement/test_training_with_third_party.py

```diff
@@ -0,0 +1,38 @@
+from pathlib import Path
+import pytest
+import os
+
+from tests.common.helpers import PROJECT_ROOT
+from tests.common.helpers import TEST_ROOT
+from tests.torch.helpers import Command
+from tests.torch.test_sanity_third_party import create_command_line
+
+
+class TestMovementWithTransformers:
+    @pytest.fixture(autouse=True)
+    def setup(self, temp_folder):
+        self.VENV_PATH = str(temp_folder["venv"])
+        self.VENV_ACTIVATE = str(". {}/bin/activate".format(self.VENV_PATH))
+        self.PYTHON_EXECUTABLE = str("{}/bin/python".format(self.VENV_PATH))
+        self.TRANSFORMERS_REPO_PATH = str(os.path.join(temp_folder["repo"], "transformers"))
+        self.CUDA_VISIBLE_STRING = "export CUDA_VISIBLE_DEVICES=0;"
+        self.PATH_TO_PATCH = str(os.path.join(PROJECT_ROOT, "third_party_integration", "huggingface_transformers",
```

Comment:
Seems like test is not finalized, is it? We also need to apply a patch with an application code of JPQD, right?

Reply:
No jpqd-related patch to transformers needed so far.  Along with 0001-Modifications-for-NNCF-usage.patch, we will only need a JPQD config file.

Reply:
Sounds great! Will built-in KD feature allow reproducing the results?
I remember some concerns about that feature. Are they valid, @vuiseng9?